src/amr/messengers/hybrid_hybrid_messenger_strategy.hpp

```diff
@@ -323,8 +309,9 @@ namespace amr
             // nodes may not have been copied correctly, due to a bug in SAMRAI
             // it seems these nodes are only on ghost box border if that border
             // overlaps an old level patch border. See https://github.com/LLNL/SAMRAI/pull/293
-            magPatchGhostsRefineSchedules[levelNumber]->fillData(initDataTime);
-            elecPatchGhostsRefineSchedules[levelNumber]->fillData(initDataTime);
+
+            // magPatchGhostsRefineSchedules[levelNumber]->fillData(initDataTime);
+            // elecPatchGhostsRefineSchedules[levelNumber]->fillData(initDataTime);
```

Comment:
If we do not switch to the samrai version https://github.com/nicolasaunai/SAMRAI/tree/feature/srcmask

we will see divb in harris on merge (likely)